scripts/check_if_submodules_head_are_ancestors_of_origin_master.sh

```diff
@@ -32,9 +31,9 @@ do
       echo "Hash HEAD:          $HASH_HEAD"
       echo "Hash origin/master: $HASH_ORIGIN_MASTER"
       echo "Log Line HEAD:"
-      echo "$(git log --oneline -n 1 HEAD)"
+      git log --oneline -n 1 HEAD
```

Comment:
Reason
 ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.

Reply:
:+1: